errors/HiddenConstructors.md

```diff
@@ -0,0 +1,23 @@
+# `HiddenConstructors` Warning
+
+## Example
+
+```purescript
+module ShortFailingExample (N) where
```

Comment:
Perhaps this should say:
```diff
- module ShortFailingExample (N) where
+ module ShortFailingExample
+  ( N -- notice how the constructor isn't exported here (i.e. `N(..)`)
+  ) where

Reply:
I’m not sure this warrants a comment, but I will add one if others feel the same. I’ve renamed the constructor and added diffs to the suggested fixes to disambiguate things though.